src/widget/textbox.rs

```diff
@@ -460,6 +474,18 @@ fn next_grapheme(src: &str, from: usize) -> usize {
     }
 }
 
+/// Checks if there is a next character from the given index.
+fn next_grapheme_exsists(src: &str, from: usize) -> bool {
```

Comment:
Typo `next_grapheme_exsists` -> `next_grapheme_exists`

Reply:
Wops thanks for catching that.